### PR TITLE
Upload output boundary

### PR DESCRIPTION
**Pull request by Sukh-04**
Updated Upload Output Boundary File

Note: Used UploadOutputData

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DrDabbidy) [inline .idea/misc.xml:1]: Is this an accident?

**Review by DrDabbidy — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (Sukh-04) [reply .idea/misc.xml:1]: Yes it was, thank you for calling it out!

**Review by jchionak — APPROVED** [review]
This is fine, I think we are going to separate the Upload and Transcribe use cases

**Review by jchionak — APPROVED** [review]

**Review by jchionak — APPROVED** [review]

**Review by jchionak — APPROVED** [review]

**Review by DrDabbidy — APPROVED** [review]
Good for now. We will have to update, but I want to get this merged.

**Pull request merged**